ONSdigital/dp-frontend-renderer
# Release 1.39.0

### What

Release 

- [x]  [Nomis dataset landing page copy updated](https://trello.com/c/PDbljF2K/468-nomis-2011-census-datasets-add-the-nomis-description)


### How to review

- Check nothing has slipped into this release that shouldn't be here
- Check that everything has been signed off

### Who can review

Anyone except me
